code/modules/mod/mod_theme.dm

```diff
@@ -1302,8 +1302,8 @@
 	siemens_coefficient = 0
 	complexity_max = DEFAULT_MAX_COMPLEXITY + 5
 	charge_drain = DEFAULT_CHARGE_DRAIN * 2
-	slowdown_inactive = 2
-	slowdown_active = 1.5
+	slowdown_inactive = 3
+	slowdown_active = 0.4
```

Comment:
nah, at max make it as fast as the normal engineering ones

Reply:
Gay

Reply:
ok

Reply:
happy pride month